docs/preview.yml

```diff
@@ -37,11 +37,12 @@ asciidoc:
     page-no-canonical: true
     page-origin-private: true
     page-hide-toc: false
+    page-mixpanel: 4bfb2414ab973c741b6f067bf06d5575
```

Comment:
what does it do?

Reply:
Good question!

It is used to track some user interactions in the page, such as switching tabs (eg in drivers), copying or running code from code blocks.